src/Dapr.Client/BulkGetItem.cs

```diff
@@ -0,0 +1,44 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Dapr
+{
+    /// <summary>
+    /// Represents a state object returned from a bulk get state operation.
+    /// </summary>
+    public sealed class GetBulkItem
```

Comment:
Can this be a readonly struct instead? this is an extra allocation per item that seems like it can be avoided.

Reply:
Change